pkg/lib/hooks.js

```diff
@@ -325,3 +325,25 @@ export function useEvent(obj, event, handler) {
 export function useInit(func, deps, comps) {
     return useObject(func, null, deps || [], comps);
 }
+
+/* XXX - document this
+ */
+
+export function useMockConfig() {
+    useEvent(window, "test_mock_changed");
+    return window.test_mock;
+}
+
+export function useMock(topic) {
+    let config = useMockConfig();
+
+    return (val, tag) => {
+        if (!config)
+            return val;
+        if (topic)
+            config = config[topic];
+        if (tag && config)
+            config = config[tag];
+        return config || "mock";
```

Comment:
These 4 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17654-20220819-071652-20323b89-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)